src/Documenter.jl

```diff
@@ -34,13 +47,13 @@ include("Writers/Writers.jl")
 include("Deps.jl")
 
 import .Utilities: Selectors
+import .Writers.HTMLWriter: HTML
 
 
 # User Interface.
 # ---------------
 
-export Deps, makedocs, deploydocs, hide
-
+export Deps, makedocs, deploydocs, hide, HTML
```

Comment:
It turns out that `HTML` is already being exported by `Base`, so one needs to do explicit `Documenter.HTML`. But that's a bit long.. so I am wondering if there is a better way.

Maybe `export` `Writers` and have it be accessible by `Writers.HTML`:

```julia
makedocs(
  Writers.HTML(prettyurls=false),
  ...
)
```

Reply:
I opted for `Documenter.HTML` because pretty soon `Writers` will only have a single writer under it. Also, `make.jl` files shouldn't be edited too often, so the extra typing seems insignificant. 

I can change it if you feel strongly that `Writers.HTML` is the right way forward.